recipe/meta.yaml

```
@@ -50,15 +52,19 @@ build:
   ignore_run_exports:
     - python *                               # [megabuild]
     - numpy *                                # [megabuild]
-  skip: true  # [win]
+
+  # TODO(baszalmstra): Remove this before merging
+  skip: true  # [not win or not (cuda_compiler_version == "12.6" or cuda_compiler == "None")]
+  skip: true  # [py!=312]
```

Think we'll have to at least remove this one
```suggestion
  skip: true  # [py!=312]
```